benchmarks/DeepLearning/Models/Bert/CMakeLists.txt

```diff
@@ -0,0 +1,167 @@
+  OUTPUT
+    ${CMAKE_CURRENT_SOURCE_DIR}/forward.mlir
+    ${CMAKE_CURRENT_SOURCE_DIR}/subgraph0.mlir
+  COMMAND
+    python3 ${CMAKE_CURRENT_SOURCE_DIR}/buddy_bert_import.py ${CMAKE_CURRENT_SOURCE_DIR}
+  COMMENT "Generating forward.mlir, subgraph0.mlir"
+)
+
+add_custom_command(
+  OUTPUT forward_auto_vectorization.o
+  COMMAND
+    cat ${CMAKE_CURRENT_SOURCE_DIR}/forward.mlir |
+    sed -e {s/@forward/@forward_auto_vectorization/}
+        -e {s/@subgraph0/@subgraph0_auto_vectorization/} |
+    ${LLVM_MLIR_BINARY_DIR}/mlir-opt 
+    -pass-pipeline
+    "builtin.module(func.func(tosa-to-linalg-named, tosa-to-linalg, tosa-to-tensor, tosa-to-arith), \
+              empty-tensor-to-alloc-tensor, convert-elementwise-to-linalg, arith-bufferize, \
+              func.func(linalg-bufferize, tensor-bufferize), func-bufferize)" |
+    ${LLVM_MLIR_BINARY_DIR}/mlir-opt
+    -pass-pipeline
+    "builtin.module(func.func(buffer-deallocation-simplification, convert-linalg-to-loops), \
+              eliminate-empty-tensors, func.func(llvm-request-c-wrappers), \
+              convert-math-to-llvm, convert-math-to-libm, convert-scf-to-cf, \
+              convert-arith-to-llvm, expand-strided-metadata, finalize-memref-to-llvm, \
+              convert-func-to-llvm, reconcile-unrealized-casts)" |
+    ${LLVM_MLIR_BINARY_DIR}/mlir-translate -mlir-to-llvmir |
+    ${LLVM_MLIR_BINARY_DIR}/llc -O3
+      -mtriple=${BUDDY_OPT_TRIPLE} -mattr=${BUDDY_OPT_ATTR} -filetype=obj
+      -o ${CMAKE_CURRENT_BINARY_DIR}/forward_auto_vectorization.o
+  DEPENDS
+    ${CMAKE_CURRENT_SOURCE_DIR}/forward.mlir
+  COMMENT "Building forward_auto_vectorization.o"
+  VERBATIM)
+
+add_custom_command(
+  OUTPUT subgraph0_auto_vectorization.o
+  COMMAND
+    cat ${CMAKE_CURRENT_SOURCE_DIR}/subgraph0.mlir |
+    sed -e {s/@subgraph0/@subgraph0_auto_vectorization/} |
+    ${BUDDY_MLIR_BINARY_DIR}/buddy-opt 
+      -pass-pipeline "builtin.module(func.func(tosa-to-linalg-named, tosa-to-linalg, tosa-to-tensor, tosa-to-arith), empty-tensor-to-alloc-tensor, convert-elementwise-to-linalg, func-bufferize-dynamic-offset, arith-bufferize, func.func(linalg-bufferize, tensor-bufferize))" |
+    ${BUDDY_MLIR_BINARY_DIR}/buddy-opt
+      -convert-elementwise-to-linalg
+      -func-bufferize-dynamic-offset
+      -arith-bufferize
+      -func-bufferize
+      -tensor-bufferize
+      -linalg-bufferize
+      -finalizing-bufferize
+      -convert-linalg-to-loops
+      -lower-affine
+      -convert-scf-to-cf
+      -llvm-request-c-wrappers
+      -convert-math-to-llvm
+      -convert-math-to-libm
+      -convert-arith-to-llvm
+      -convert-func-to-llvm
+      -expand-strided-metadata
+      -finalize-memref-to-llvm
+      -reconcile-unrealized-casts |
+    ${LLVM_MLIR_BINARY_DIR}/mlir-translate -mlir-to-llvmir |
+    ${LLVM_MLIR_BINARY_DIR}/llc -O3
+      -mtriple=${BUDDY_OPT_TRIPLE} -mattr=${BUDDY_OPT_ATTR} -filetype=obj
+      -o ${CMAKE_CURRENT_BINARY_DIR}/subgraph0_auto_vectorization.o
```

Comment:
Translation from .ll to .o should be updated to clang. See the following examples:
```
${LLVM_MLIR_BINARY_DIR}/mlir-translate -mlir-to-llvmir -o forward_scalar.ll
  COMMAND ${LLVM_MLIR_BINARY_DIR}/clang -O3 ${CLANG_FLAGS_LIST} forward_scalar.ll
    -c -save-temps -o ${CMAKE_CURRENT_BINARY_DIR}/forward_scalar.o
```

Current implementation will lead to compilation failure due to lack of compiler flag "no-pie"  in latest version. Please fix it in another PR.